tests/results/DiffTest.py

```diff
@@ -37,13 +37,15 @@ def test_delete_line(self):
         additions, deletions = self.uut.stats()
         self.assertEqual(deletions, 1)
         self.assertRaises(ValueError, self.uut.delete_line, 0)
+        self.assertRaises(ValueError, self.uut.delete_line, 10)
 
     def test_delete_lines(self):
-        self.uut.delete_lines(1, 10)
-        self.uut.delete_lines(10, 20)
+        self.uut.delete_lines(1, 2)
+        self.uut.delete_lines(2, 3)
         additions, deletions = self.uut.stats()
-        self.assertEqual(deletions, 20)
-        self.assertRaises(ValueError, self.uut.delete_lines, 0, 10)
+        self.assertEqual(deletions, 3)
+        self.assertRaises(ValueError, self.uut.delete_lines, 0, 2)
+        self.assertRaises(ValueError, self.uut.delete_lines, 1, 6)
```

Comment:
Wait, all these should be IndexError, not ValueError. How are the tests even passing?